.gitignore

```diff
@@ -22,3 +22,5 @@ work
 build/
 pom.xml.releaseBackup
 release.properties
+kubectl
```

Comment:
Let's do not add the Kubernetes integration tests in here. Spring Vault is already verified and that way we reduce build complexity.